app/controllers/admin/users_controller.rb

```diff
@@ -2,7 +2,13 @@ class Admin::UsersController < Admin::BaseController
 
   def index
     authorize(User)
-    @users = User.all.page(params[:page])
+    @users = User.member.page(params[:page])
+  end
+
+  def index_admins
+    authorize(User)
+    @users = User.admin.page(params[:page])
+    render :index
   end
 
```

Comment:
It's what rails_enum provides, can change to be plural if you'd like (using aliases)